packages/integration-tests/package.json

```diff
@@ -19,5 +19,9 @@
     "node-fetch": "^2.6.0",
     "promise-retry": "^1.1.1",
     "web3": "1.0.0-beta.36"
+  },
+  "devDependencies": {
+    "truffle": "^4.1.14",
+    "mocha": "^5.2.0"
```

Comment:
i think we should keep devDeps in root package.json. should fall back to it. also we should call test from root as mocha is already there

Reply:
that case, you cannot install only integration-tests and runs it. for example 
`npm install @omisego/integration-tests` because it doesn't have `mocha` 

the dependencies on developments will resolve nicely with lerna bootstrap